configs/retinanet/README.md

```diff
@@ -24,3 +24,22 @@
 | X-101-32x4d-FPN | pytorch |   2x    |    -     |       -        |  40.1  | [model](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/retinanet/retinanet_x101_32x4d_fpn_2x_coco/retinanet_x101_32x4d_fpn_2x_coco_20200131-237fc5e1.pth) &#124; [log](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/retinanet/retinanet_x101_32x4d_fpn_2x_coco/retinanet_x101_32x4d_fpn_2x_coco_20200131_114812.log.json) |
 | X-101-64x4d-FPN | pytorch |   1x    |   10.0   |      8.7       |  41.0  | [model](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/retinanet/retinanet_x101_64x4d_fpn_1x_coco/retinanet_x101_64x4d_fpn_1x_coco_20200130-366f5af1.pth) &#124; [log](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/retinanet/retinanet_x101_64x4d_fpn_1x_coco/retinanet_x101_64x4d_fpn_1x_coco_20200130_003008.log.json) |
 | X-101-64x4d-FPN | pytorch |   2x    |    -     |       -        |  40.8  | [model](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/retinanet/retinanet_x101_64x4d_fpn_2x_coco/retinanet_x101_64x4d_fpn_2x_coco_20200131-bca068ab.pth) &#124; [log](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/retinanet/retinanet_x101_64x4d_fpn_2x_coco/retinanet_x101_64x4d_fpn_2x_coco_20200131_114833.log.json) |
+
+### Some tricks
+
+We also support exponential momentum average (EMA) over the number of positive samples as loss normalizer to improve the performance. This trick is introduced in [Detectron2](https://github.com/facebookresearch/detectron2/blob/bf11a9b5bbc8e80cfe713c547203576a77a09065/detectron2/modeling/meta_arch/retinanet.py#L108) to stabilize the training by reducing the variance of foreground number. Adding the following two lines in the head config could enable this option.
+
+```python
+loss_normalizer_momentum=0.9,
+```
+
+To enable a fair comparison when benchmarking with Detectron2, we use it in the benchmark configs. However, we do not recommend it as a default option since so many methods have proposed based on the original baseline. We compare the performance of models before/after adding EMA loss normalizer in the following table.
```

Comment:
We actually didn't observe significant improvements from this option (maybe at most 0.2AP). So we feel comfortable adding it by default and consider it as a bugfix to the original poorly design normalizer (it can fix the poor performance of RetinaNet in 1im/GPU case).

I guess the improvements listed in this table may come from the original incorrect impl. pointed out at https://github.com/open-mmlab/mmdetection/pull/2780#discussion_r428435677 , which effectively changed the scale of loss.

Reply:
Yes, this option indeed only brings about 0.2~0.3 AP improvement. The [bug version](https://github.com/open-mmlab/mmdetection/pull/2780#discussion_r428435677) actually brings more gains but we are not going to use that since it is not very reasonable and may not be generalizable to other models.